Sources/SmokeFrameworkApplicationGenerate/main.swift

```diff
@@ -106,15 +109,29 @@ private func startCodeGeneration(
         applicationDescription: applicationDescription,
         applicationSuffix: applicationSuffix)
     
-    return try SmokeFrameworkCodeGeneration.generateFromModel(
-        modelFilePath: modelFilePath,
-        modelType: SwaggerServiceModel.self,
-        generationType: generationType,
-        customizations: customizations,
-        applicationDescription: fullApplicationDescription,
-        operationStubGenerationRule: operationStubGenerationRule,
-        initializationType: initializationType,
-        modelOverride: modelOverride)
+    if version == 2 {
+        return try SmokeFrameworkCodeGeneration.generateFromModel(
+            modelFilePath: modelFilePath,
+            modelType: SwaggerServiceModel.self,
+            generationType: generationType,
+            customizations: customizations,
+            applicationDescription: fullApplicationDescription,
+            operationStubGenerationRule: operationStubGenerationRule,
+            initializationType: initializationType,
+            modelOverride: modelOverride)
+    } else if version == 3 {
+        return try SmokeFrameworkCodeGeneration.generateFromModel(
+            modelFilePath: modelFilePath,
+            modelType: OpenAPIServiceModel.self,
+            generationType: generationType,
+            customizations: customizations,
+            applicationDescription: fullApplicationDescription,
+            operationStubGenerationRule: operationStubGenerationRule,
+            initializationType: initializationType,
+            modelOverride: modelOverride)
+    } else {
+        fatalError("Invalid version")
```

Comment:
I think if no version is specified in the command we should just generate `SwaggerServiceModel` code instead of throwing an error.

Reply:
Wouldn't that cause issues if the swagger.yaml is openapi 3.0 version though? Or are we okay with having the user assume the default is 2.0?

Reply:
Yeah I don't think the version should be a required part of the command, I think the version command value should be optional.

Reply:
Yeah got it, done.